lldb/test/Shell/Expr/TestObjCIDCast.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: %clangxx_host %s -g -o %t
+//
+// RUN: %lldb -f %t \
+// RUN:  -o "settings set interpreter.stop-command-source-on-error false" \
+// RUN:  -o "b main" -o run -o "expression --language objc -- *(id)0x1234" -o exit 2>&1 | FileCheck %s
```

Comment:
Does this test work on Linux?
Is there another constant that is even less likely to ever be a real pointer?
Would 0x00000001 work?

Reply:
done